modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/transformer/hdbdd/HdbddTransformer.java

```diff
@@ -377,6 +383,41 @@ private XSKDataStructureHDBTableColumnModel transformFieldSymbolToColumnModel(Fi
     return columnModel;
   }
 
+  /**
+   * @param fieldSymbol: fieldSymbol
+   */
+  private XSKDataStructureHDBTableCalculatedColumnModel transformFieldSymbolToCalculatedColumnModel(FieldSymbol fieldSymbol) {
+    XSKDataStructureHDBTableCalculatedColumnModel calculatedColumnModel = new XSKDataStructureHDBTableCalculatedColumnModel();
+
+    calculatedColumnModel.setColumnName(fieldSymbol.getName());
+
+    if (fieldSymbol instanceof EntityElementSymbol) {
+      EntityElementSymbol elementSymbol = (EntityElementSymbol) fieldSymbol;
+      calculatedColumnModel.setStatement(elementSymbol.getStatement());
+    }
+
+    if (fieldSymbol.getType() instanceof BuiltInTypeSymbol) {
```

Comment:
Could you move the implementation of these if statements into separate private methods?